sdk/mediaservices/Azure.Media.Analytics.Edge/tests/Samples/LiveVideoAnalyticsSample.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using Azure.Core.TestFramework;
+using Azure.Identity;
+using Azure.Media.Analytics.Edge.Models;
+using Microsoft.Azure.Devices;
+using NUnit.Framework;
+using System.Text.Json;
+using Newtonsoft.Json.Linq;
+using System.Threading.Tasks;
+using System.IO;
+using System.Text;
+using Azure.Core;
+using System.Linq;
+
+namespace Azure.Media.Analytics.Tests.Samples
+{
+    public class LiveVideoAnalyticsSample
```

Comment:
Can we remove this file if we are not adding sample here?

Reply:
Without any tests the PR fails some checks. I added a comment in this file to better explain why it's needed.